pandas/tests/indexing/test_categorical.py

```diff
@@ -758,3 +758,23 @@ def test_map_with_dict_or_series(self):
         output = cur_index.map(mapper)
         # Order of categories in output can be different
         tm.assert_index_equal(expected, output)
+
+    def test_operations_with_interval_categories_index(self):
+        ind = pd.CategoricalIndex(pd.interval_range(start=0.0, end=2.0))
+        df = pd.DataFrame([[1, 2]], columns=ind)
+        # __add__
```

Comment:
You might be able to parametrize over the different operations, and there might be some existing fixtures that do this.  Probably best to see where this gets moved and follow any conventions for existing tests of this nature.

Reply:
Yes, fixture can make this code shorter and get the ops abstraction. Let me know where I should put the test to. Thanks~